nat.go

```diff
@@ -36,6 +36,9 @@ type Config struct {
 	UseInterfaces []string
 	// Blacklist given addresses for ICE negotiation.
 	BlacklistAddresses []*net.IPNet
+	// TOS, if >0, sets IP_TOS to this value. Note an error is consider
```

Comment:
Typos:

consider -> considered
if is -> it is

Reply:
Done